File: ProjectManager/Models/Comment.cs
using System;
namespace ProjectManager.Models
{
    public class Comment
    {
        public int CommentId { get; set; }

        public string CreationTime { get; set; }
        public Task Task { get; set; }
        public string Text { get; set; }
        public User User { get; set; }
    }
}

File: ProjectManager/wwwroot/js/ajax/comment.js
/** Adds a new comment to the database
 * 
 * @param {string} text The text of the comment
 * @param {number} taskId The ID of the task the comment belongs to
 * @param {number} userId The ID of user who made the comment
 */
function addCommentToDatabase(text, taskId, userId) {
    $.ajax({
        type: "POST",
        url: "/Comment/New",
        data: { text: text, taskId: taskId, userId: userId }
    });
}